osbs/utils/__init__.py

```diff
@@ -601,7 +601,7 @@ def user_warning_log_handler(self, message):
         'message': message,
     }
     msg = json.dumps(content)
-    self._log(USER_WARNING_LEVEL, msg)
+    self._log(USER_WARNING_LEVEL, msg, None)
```

Comment:
Shouldn't be this an empty list instead of None?

Reply:
It doesn't matter, `args` apply into record only when they are defined, because it's None, it never will be used (see [code ](https://github.com/python/cpython/blob/28ad12f8fe889a741661eb99daacebd9243cc1ba/Lib/logging/__init__.py#L359) where it applies)

Reply:
ok